XIVComboPlugin/Configuration/CustomComboPreset.cs

```diff
@@ -48,7 +48,7 @@ public enum CustomComboPreset : long
         [CustomComboInfo("Mythril Tempest Combo", "Replace Mythril Tempest with its combo chain", 21)]
         WarriorMythrilTempestCombo = 1L << 10,
 
-        [CustomComboInfo("IR to Primal Rend", "Replace Inner Release with Primal Rend when Primal Rend Ready", 21)]
+        [CustomComboInfo("IR Combo", "Replace Inner Release with Primal Rend or Primal Ruination if they are ready and Wrathful status is not active", 21)]
```

Comment:
```suggestion
        [CustomComboInfo("IR Combo", "Replace Inner Release with Primal Rend or Primal Ruination when ready and not Wrathful", 21)]
```

I don't like this combo tons, but this maintains conceptual parity with GNB's Continuation combos. Due to the nature of checking buffs, I think a little bit of responsiveness is lost. But overall it's fine.